<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>卡密认证</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>欢迎使用方舟：生存飞升 报警器 - 请进行卡密认证</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请输入卡密：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cdkEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="validateButton">
     <property name="text">
      <string>验证</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
